problemsets-mat223/modules/module14.tex

```diff
@@ -122,13 +129,13 @@
 	Let $\mathcal T:\R^2\to\R^2$ be defined by $\mathcal T\mat{x\\y}=\matc{2x+y\\-x+\tfrac{1}{2}y}$.
 	Find $\det(\mathcal T)$.
 
-	XXX Finish (this is the example from earlier)
+	Refer to the example from earlier, $\mathcal T$ is orientation preserving. Therefore, $\det(\mathcal T)=2$.
 \end{example}
 \begin{example}
 	Let $\mathcal S:\R^2\to\R^2$ be defined by $\mathcal S\mat{x\\y}=\mat{-x+y\\x+y}$.
 	Find $\det(\mathcal T)$.
 
-	XXX Finish (this reverses orientation)
+	Similarly to the earlier example, we compute $\mathcal S\mat{1\\0} = \mat{-1\\1}$ and $\mathcal S\mat{0\\1} = \mat{1\\1}$. Then, Plot them in the graph and we may notice that the orientation of $\mathcal S$ is reversing. Thus, $\det(\mathcal S) = - \Vol(\mathcal S(C_2)) = -2$.
```

Comment:
`\mathcal S\mat{1\\0}` -> `\mathcal S\xhat`, etc.
`Then, Plot them ...` ->`Plotting $\mathcal S\xhat$ and $\mathcal S\yhat$, we see that $\mathcal S$ is orientation reversing.`